### PR TITLE
Fix `tsc:fast`

### PR DESCRIPTION
**Pull request by curtisman**
Change `fluid-build` `-s` switch to `-t`

### CODE REVIEW

File: .vscode/tasks.json [hunk 1]
```diff
@@ -1,103 +1,97 @@
...
+		{
+			"label": "Build Current Tests",
+			"type": "process",
+			"command": "node",
+			"args": [
+				"${workspaceRoot}/node_modules/@fluidframework/build-tools/dist/fluidBuild/fluidBuild.js",
+				"--root",
+				"${workspaceRoot}",
+				"--vscode",
+				"-t",
```
Comment (curtisman) [inline .vscode/tasks.json:37]: This is the fix.  The rest is just changing to tabs (from auto fix on save with prettier)

File: README.md
Comment (tylerbutler) [inline README.md:1]: I see similar changes needed in:

- packages\tools\devtools\devtools-core\README.md
- packages\tools\devtools\devtools\README.md
- packages\tools\devtools\devtools-core\README.md
- packages\tools\devtools\devtools-view\README.md
- azure\package.json

Comment (curtisman) [reply README.md:1]: Added except `azure\package.json`, which will be deleted in PR#15861

**Review by tylerbutler — APPROVED** [review]
📈

**Pull request merged**